converter/src/main/java/gov/cms/qpp/conversion/model/error/Detail.java

```diff
@@ -21,7 +54,7 @@
 	private String type;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1213)